docs/source/quick_start/first_run.rst

```diff
@@ -0,0 +1,163 @@
+    ├── versions/
+    ├── world/
+    ├── banned-ips.json
+    ├── banned-players.json
+    ├── eula.txt
+    ├── ops.json
+    ├── server.jar
+    ├── server.properties
+    ├── usercache.json
+    └── whitelist.json
+
+With this command to start it:
+
+.. code-block:: bash
+
+    java -Xmx1024M -Xms1024M -jar server.jar nogui
+
+You can also use other server software, such as Fabric, Spigot, Paper, etc.
+
+.. note::
+
+    Configure and troubleshoot your server software **BEFORE** you introduce MCDR. Otherwise, you can't distinguish whether a problem is caused by MCDR or your server software
+
+    MCDR was born as a daemon for server softwares of Minecraft: Java Edition. Most plugins are predicated on that. If you want to run MCDR with a Bedrock Edition server, consider using Java Edition server software with `GeyserMC <https://geysermc.org/>`__ to provide Bedrock Edition compatibility
+
+    Of course, if you don't need the MCDR plugin community, you can run anything you want with MCDR
+
+Initialize
+----------
+
+After MCDR has been installed, you can verify the installation with the following command:
+
+.. code-block-mcdr-version:: bash
+
+    $ mcdreforged
+    MCDReforged v@@MCDR_VERSION@@
+
+Let's say your are going to start MCDR in a folder named ``my_mcdr_server``. Then you can run the following commands to initialize the environment for MCDR:
+
+.. code-block:: bash
+
+    cd my_mcdr_server
+    mcdreforged init
+
+MCDR will generate its default structure like this:
+
+.. code-block::
+
+    my_mcdr_server/
+     ├─ config/
+     ├─ logs/
+     │   └─ MCDR.log
+     ├─ plugins/
+     ├─ server/
+     ├─ config.yml
+     └─ permission.yml
+
+Animated demo:
+
+.. asciinema:: resources/init.cast
+    :rows: 10
+
+|
+
+Configure
+---------
+
+Server Software
+~~~~~~~~~~~~~~~
+
+Remember the server software you have prepared? Put it into the ``server`` folder. The directory structure should be something like this:
+
+.. code-block:: diff
+
+        my_mcdr_server/
+        ├─ config/
+        ├─ logs/
+        │   └─ MCDR.log
+        ├─ plugins/
+        ├─ server/
+    ++  │   ├─ ...
+    ++  │   ├─ server.jar
+    ++  │   └─ server.properties
+        ├─ config.yml
+        └─ permission.yml
+
+Config File
+~~~~~~~~~~~
+
+Then, edit the ``config.yml`` file to configure MCDR. You can find more information about this in :doc:`/configuration`
+
+For most users, there are 4 parts of the configuration that you need to be aware of
+
+- :ref:`configuration:language`: the language using in MCDR
+- :ref:`configuration:start_command`: how MCDR starts your server
+- :ref:`configuration:handler`: how MCDR read your server outputs
+- :ref:`configuration:encoding, decoding`: how MCDR handles the server IO stream
+
+Read each section carefully and make sure you filled in the correct values.
+
+As a simple example, this is all you need to aware for a vanilla Minecraft 1.21 server with Java 21:
+
+.. code-block:: yaml
+
+    language: en_us
```

Comment:
`language` is not required here

Reply:
it's listed above for zh_cn users
```rst
- :ref:`configuration:language`: the language using in MCDR
```
so here it is for consistency

Reply:
"this is all you need to aware for a vanilla Minecraft 1.21 server with Java 21", and `langage` is not necessary for correctly running a vanilla Minecraft 1.21 server with MCDR

Reply:
sounds fair